pkg/cmd/pr/create/create.go

```diff
@@ -196,6 +205,7 @@ func NewCmdCreate(f *cmdutil.Factory, runF func(*CreateOptions) error) *cobra.Co
 	fl.StringVarP(&opts.BaseBranch, "base", "B", "", "The `branch` into which you want your code merged")
 	fl.StringVarP(&opts.HeadBranch, "head", "H", "", "The `branch` that contains commits for your pull request (default: current branch)")
 	fl.BoolVarP(&opts.WebMode, "web", "w", false, "Open the web browser to create a pull request")
+	fl.BoolVarP(&opts.AutofillVerbose, "fillverbose", "", false, "Use commits msg+body for description")
```

Comment:
Using hyphens to separate words like `fill-first`:

```suggestion
	fl.BoolVarP(&opts.AutofillVerbose, "fill-verbose", "", false, "Use commits msg+body for description")
```

Reply:
Nice, done